port/blueprint/model.go

```diff
@@ -64,6 +64,8 @@ type StringItems struct {
 	Format  types.String `tfsdk:"format"`
 	Default types.List   `tfsdk:"default"`
 	Pattern types.String `tfsdk:"pattern"`
+	Enum    types.List   `tfsdk:"enum"`
```

Comment:
did you run `gofmt` on this file? indent looks a bit strange (IIRC your IDEush should do that automagically)